socorro/external/es/crashstorage.py

```diff
@@ -17,6 +18,10 @@
 from socorro.external.crashstorage_base import Redactor
 
 
+FIELD_NAME_STRING_ERROR_REGEX = re.compile(r'field=\"([\w.]+)\"')
```

Comment:
Well, ``\w`` catches everything we care about except hyphens. We do have a bunch a very weird field names, but I don't think we actually want to capture these. I think it's a risk that we might mess the regex up and end up with some weird string matching the pattern.